public/app/features/alerting/unified/AlertGroups.tsx

```diff
@@ -19,7 +21,8 @@ import { getFiltersFromUrlParams } from './utils/misc';
 import { initialAsyncRequestState } from './utils/redux';
 
 const AlertGroups = () => {
-  const [alertManagerSourceName] = useAlertManagerSourceName();
+  const alertManagers = useAlertManagerSources('instance');
```

Comment:
Suggestion: This function call signature was somewhat confusing to me at first, maybe we should either rename the hook to `useAlertManagerSourcesByPermission` or change the function signature to pass an object literal like `{ permission: 'instance' }`.